### DISCUSSION TITLE
DHCP crashes my home network

### DISCUSSION THREAD

**Discussion by Ragetti13** [AdguardTeam/AdGuardHome#3183] [Q&A]
Hi, 

I have AdGuard Home setup on my HomeAssistant (running on a raspberry pi4). 

The adblocker runs perfectly well when my home router DNS router is set to the IP of the pi but when the router DHCP server is turned off and Adguard is set as the DHCP server no devices can reach the internet.

```
 Add-on version: 4.1.6
 You are running the latest version of this add-on.
 System: Home Assistant OS 5.13  (aarch64 / raspberrypi4-64)
 Home Assistant Core: 2021.5.5
 Home Assistant Supervisor: 2021.04.3
```

Does anyone have any suggestions on what the issue could be?

James

**Comment by ameshkov** [comment]
To troubleshoot this issue we need to see AdGuard Home logs.

1. [Configure](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog) AdGuard Home to write verbose-level log.
2. Reproduce the issue.
3. Post the log file here.

**Reply by Ragetti13** [reply]
Hi,

Thanks for the reply but i think i managed to get this working.

I had to re-install the AdGuard add-on via supervisor and then assign the static IP again which seemed to work.

I will report back if anything else changes.

James
